docs/website/src/site/styles/homepage/call-to-action.scss

```diff
@@ -0,0 +1,311 @@
+@import './../breakpoints';
+
+$transition-timing: 200ms;
+
+.ds-call-to-action {
+  background: var(--cta-background);
+  border-radius: 1.2rem;
+  direction: ltr;
+  display: grid;
+  gap: 4rem;
+  grid-template-columns: none;
+  grid-template-rows: 1fr auto;
+  justify-items: center;
+  margin: 3.2rem 0;
+  overflow: hidden;
+  padding: 4rem 3.2rem;
+
+  @media screen and (min-width: $md-lg-breakpoint) {
+    direction: rtl;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: none;
+    padding: 4rem;
+  }
+
+  @media screen and (min-width: $lg-breakpoint) {
+    direction: ltr;
```

Comment:
why? this is the writing direction, that shouldn't change, right?

Reply:
It was a kind of workaround to change order in css grid, but yeah I'll use `order` instead :)